### PR TITLE
Fix #200: Add contribute.json

### LINKED ISSUE #200: Add a contribute.json
Mozilla says that this file should be served up for all websites.

https://infosec.mozilla.org/guidelines/web_security#contributejson

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

**Comment by openjck** [comment]
@pdehaan It's not letting me add you as a reviewer, perhaps because you have authorship of one of the commits, but I'd appreciate any feedback.

File: public/contribute.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+    "name": "Ensemble",
+    "description": "Ensemble is the platform that powers the Firefox Public Data Report (https://data.firefox.com), a weekly public report on the activity, behavior, and hardware configuration of Firefox Desktop users.",
```
Comment (pdehaan) [inline public/contribute.json]: Do you want to remove "Desktop" here and be vague? Or we can leave it in, and maybe remove it in the future (memories permitting) when we add Mobile metrics?

File: public/contribute.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        "node",
+        "react",
+        "react-refetch"
+    ],
+    "bugs": {
+        "list": "https://github.com/mozilla/ensemble/issues",
+        "report": "https://github.com/mozilla/ensemble/issues/new",
+        "mentored": "https://github.com/mozilla/ensemble/labels/help%20wanted"
+    },
+    "urls": {
```
Comment (pdehaan) [inline public/contribute.json]: Do we not have a dev server anymore that gets rebuilt on each deploy?

**Comment by pdehaan** [comment]
> @pdehaan It's not letting me add you as a reviewer, perhaps because you have authorship of one of the commits, but I'd appreciate any feedback.

I can't seem to edit your comments or adjust labels/milestones/etc either.
You may need to add me as a repo collaborator in order to assign me as a reviewer. 🤷‍♀️

**Comment by openjck** [comment]
> I can't seem to edit your comments or adjust labels/milestones/etc either.
> You may need to add me as a repo collaborator in order to assign me as a reviewer. 🤷‍♀️

Huh... sorry about that. Maybe read permissions *do* do something even when the repo is public. I've added you back.

File: public/contribute.json (hunk 2 above)
Comment (openjck) [reply public/contribute.json]: Not currently, but Blake is working on it.

File: public/contribute.json (hunk 1 above)
Comment (openjck) [reply public/contribute.json]: +1 for vague. Good catch!

**Comment by openjck** [comment]
Updated!

**Pull request merged**